lib/fizzy/parser_expr.cpp

```diff
@@ -389,32 +404,32 @@ parser_result<Code> parse_expr(const uint8_t* pos, const uint8_t* end, uint32_t
 
         case Instr::block:
         {
-            std::optional<ValType> type;
-            std::tie(type, pos) = parse_blocktype(pos, end);
+            std::optional<ValType> block_type;
```

Comment:
Why do you need this refactoring btw?

Reply:
`type` would shadow the new variable in the outer scope (which I called `types` originally, but @chfast suggested to rename)